app/posts/ecf-v2/capturing-sit-information.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Capturing information about the school's induction tutor (SIT)
+description: "We've designed journeys to capture information about who the school's induction tutor is, and to confirm whether any changes have been made since we were last notified."
+date: 2025-11-25
+---
+
+## How we used to collect SIT information in ECF1
+
+In the existing ECF1 service, schools had to supply the name and email address for their induction tutor (SIT) before accessing the service. We used those details to send a link to the SIT and grant them access to register new early career teachers and mentors.
+
+This approach assumed the SIT would be the only person from the school who needed to use the service, and we didn't allow for multiple users. In practice, more than one person might need access, and while the SIT is the main ECTE contact, they are not always the person registering new early career teachers and mentors.
```

Comment:
Can we link to this design history too btw? https://teacher-cpd.design-history.education.gov.uk/ecf-v2/allowing-multiple-school-accounts/